packages/asset/test/AssetCreate.test.ts

```diff
@@ -1101,6 +1120,32 @@ describe('AssetCreate (/packages/asset/contracts/AssetCreate.sol)', function ()
           )
         ).to.be.revertedWith('ERC1155: burn amount exceeds totalSupply');
       });
+      it('should revert when minting tier 0 assets using createMultipleAssets', async function () {
+        const {
+          mintMultipleAssets,
+          generateMultipleMintSignature,
+          mintCatalyst,
+          metadataHashes,
+          user,
+        } = await runCreateTestSetup();
+        await mintCatalyst(4, 1);
+        const signature = await generateMultipleMintSignature(
+          user.address,
+          [0, 4],
+          [1, 1],
+          [true, true],
+          metadataHashes
+        );
+        await expect(
+          mintMultipleAssets(
+            signature,
+            [0, 4],
+            [1, 1],
+            [true, true],
+            metadataHashes
+          )
+        ).to.be.revertedWith('ERC1155: burn amount exceeds totalSupply');
```

Comment:
This is technically correct because you didn't try to mint any tier 0 cats in the fixture.
Have we got tests for minting tier 0 cats? If not, please add - we should show that minting tier 0 cats are prevented by the cat contract

Reply:
Added a test in the catalyst tests for tier 0 minting. 👍